data/frontier jobs.txt

```diff
@@ -266,7 +266,7 @@ mission "Small Pirate gambling"
 				decline
 
 			label call
-			`	"You confront them on their blatant cheating when one of them yells out, We're just playin' a game here, see? You'd better shut your big mouth if you know what's good for you," the meanest-looking one growls at you. "We can make this ugly real fast."`
+			`	You confront them on their blatant cheating when one of them yells out, "We're just playin' a game here, see? You'd better shut your big mouth if you know what's good for you!" The meanest-looking one growls at you. "We can make this ugly real fast."`
```

Comment:
Perhaps instead of just comma to exclamation point, move`The meanest-looking one growls at you.` to the front of the quote.
```suggestion
			`	You confront them on their blatant cheating when the meanest-looking one growls at you, "We're just playin' a game here, see? You'd better shut your big mouth if you know what's good for you! We can make this ugly real fast."`
```